model/gen-ai/events.yaml

```
@@ -14,6 +14,26 @@ groups:
     brief: >
       This event describes the instructions passed to the GenAI system inside the prompt.
```

```suggestion
      This event describes the system instructions passed inside the prompt to the GenAI model.
```
Reading through the markdown beginning to end, I realized the three message descriptions could be more precise.

---

thanks! I want to avoid "passed inside the prompt" since we should be able to reuse the same messages for something like OpenAI assistants, which can take instructions at the assistant creation time. I'm taking the rest though